custom/conf/app.ini.sample

```diff
@@ -110,6 +110,8 @@ SIGNING_KEY = default
 ; by setting the SIGNING_KEY ID to the correct ID.)
 SIGNING_NAME =
 SIGNING_EMAIL =
+; Sets the default trust model for repositories. Options are: collaborator, committer, collaboratorcommitter
```

Comment:
`collaboratorcommitter` is a rather lengthy and ugly name

Reply:
I'm open to suggestions...

Reply:
The alternatives I could think of would be:
- `both`
- Refactoring `DEFAULT_TRUST_MODEL` to accept comma-delimited list, such as `collaborator, commiter`
- `collabcommiter`

Reply:
* both . That's going to collide with future trust models: in particular pubkey trust models
* comma-delimited list - will collide with pubkey trust models in the future. I had considered making a flag system but it's not really flaggable at present. 
* collabcommitter - that's even worse.